carWarGame.py

```diff
@@ -51,14 +52,19 @@ def shuffle(self) -> None:
         """Shuffles the deck of cards"""
         random.shuffle(self.cards)
 
+    def split(self) -> tuple:
+        """Split cards in two equal parts"""
+        return self.cards[: self.length // 2], self.cards[self.length // 2 :]
+
     def __str__(self) -> str:
-        return f"{len(self.cards)}"
+        return self.length
```

Comment:
After my first suggestion you could do something like:


```suggestion
        return len(self)
```

Reply:
Already implemented.